test/models/notice_test.rb

```diff
@@ -1,7 +1,37 @@
-require 'test_helper'
+require './test/test_helper'
 
 class NoticeTest < ActiveSupport::TestCase
   # test "the truth" do
   #   assert true
   # end
+  test "current scope returns correct days" do
+    notice = Notice.create(start_date: Date.yesterday, end_date: Date.tomorrow) #good
+    notice2 = Notice.create(start_date: Date.yesterday, end_date: 5.days.from_now) #good
+    notice3 = Notice.create(start_date: Date. yesterday, end_date: 3.days.from_now) #good
+    notice4 = Notice.create(start_date: 2.days.ago, end_date: Date.tomorrow) #good
+
+    assert_equal(Notice.current,[notice, notice2, notice3, notice4])
+  end
+
+  test "current scope does not return reversed dates" do
```

Comment:
Making a note here that it should not be possible to create these. This should be resolved in #7